<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TiezhiUIClass</class>
 <widget class="QWidget" name="TiezhiUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="tiezhiWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>230</height>
    </rect>
   </property>
   <widget class="QPushButton" name="tieYuanhuaBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/beauty_original);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tie1Btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>15</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/sticker_aral);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tie2Btn">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>15</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/sticker_black_cat_ears);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tie3Btn">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>15</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/sticker_bud_glasses);
border-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tie7Btn">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>80</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/sticker_YYbear);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tie5Btn">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>80</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/sticker_seeking_attention);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tie4Btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/sticker_devil_gauze_mask);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tie6Btn">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>80</y>
      <width>47</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/beauty/sticker_sunglasses);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../joy.qrc"/>
 </resources>
 <connections/>
</ui>
